### DISCUSSION TITLE
Cannot import plugins, throws "Module not found: Can't resolve 'yet-another-react-lightbox/plugins/thumbnails'" error

### DISCUSSION THREAD

**Discussion by ukesh977** [igordanchenko/yet-another-react-lightbox#101] [Q&A] [answered]
Tried to apply Thumbnails plugin, but throws module not found error on compile. 
![image](https://user-images.githubusercontent.com/26924856/228744989-c76b4f80-9cf1-4d9a-9e02-78ad6a7e1e5b.png)

![image](https://user-images.githubusercontent.com/26924856/228745208-5f9a5a11-9e5f-426b-9dbd-cfcb9c1fb295.png)

![image](https://user-images.githubusercontent.com/26924856/228745365-b8ac4823-00a7-4569-b75d-df9495748cb3.png)

**Comment by igordanchenko** [comment, marked as answer]
Are you using CRA 4? CRA 4 uses an old webpack version that doesn't support package exports map. Upgrading to CRA 5 would address this. [Over here](https://github.com/igordanchenko/yet-another-react-lightbox/discussions/61#discussioncomment-4797987) you can find some workarounds for CRA 4, but upgrading the tech stack is always the right way to go.
